java/client/src/main/java/glide/api/commands/GenericClusterCommands.java

```diff
@@ -152,6 +154,109 @@ public interface GenericClusterCommands {
      */
     CompletableFuture<String> randomKey();
 
+    /**
+     * Incrementally iterates over the keys in the Cluster.
+     *
```

Comment:
let's reorganize this javadoc.  We've mangled cursor best practices in with the description of the command.  I'd suggest: 

```
     * <p>This command is similar to the SCAN command, but it is designed to work in a Cluster
     * environment. The main difference is that this command uses a {@link ClusterScanCursor} object
     * to manage iterations. For more information about the Cluster Scan implementation, see <a
     * href="https://github.com/aws/glide-for-redis/wiki/General-Concepts#cluster-scan">Cluster
     * Scan</a>.
     * <p>As with the SCAN command, the method can be used to iterate over all the keys in the database
     * that were available from the time the scan started until the scan finishes (that is, {@link ClusterScanCursor#isFinished()} returns true).
     * <p>This method guarantees that all keyslots available when the first SCAN is called will be
     * scanned before the cursor is finished. Any keys added after the initial scan request is made
     * are not guaranteed to be scanned.
     * <p>Note that the same key may be returned in multiple scans iterations.
     * 
     * <p>How to use the {@link ClusterScanCursor}: <br>
     * For each iteration, the previous scan {@link ClusterScanCursor} object should be used to continue the scan.
     * Using the same cursor object for multiple iterations may result in the same keys returned or unexpected
     * behavior. 
     * <p>When the cursor is no longer needed, call {@link ClusterScanCursor#releaseCursorHandle()} to immediately free resources tied
     * to the cursor. Note that this makes the cursor unusable in subsequent calls to scan.
```